server/utils/getClaimData.js

```diff
@@ -22,7 +22,7 @@ module.exports = async (data) => {
   let lbrynetFileExt = null;
   if (!data.fileExt) {
     lbrynetClaimResult = await getClaim(lbrynetUri).catch(() => { return 'invalid URI' });
-    lbrynetFileExt = lbrynetClaimResult && lbrynetClaimResult.file_name.split('.').slice(-1).pop();
+    lbrynetFileExt = lbrynetClaimResult && lbrynetClaimResult.file_name && lbrynetClaimResult.file_name.split('.').slice(-1).pop();
```

Comment:
I'd like to see more verbose logic, I'm not a fan of this being able to return `null` / `false`.
I'll investigate how we should handle this at a higher level (larger refactor-type issue)